ipaserver/install/server/replicainstall.py

```diff
@@ -1391,7 +1391,14 @@ def install(installer):
     dsinstance.create_ds_user()
 
     try:
-        conn.connect(ccache=ccache)
+        conn.connect(bind_dn=ipaldap.DIRMAN_DN, bind_pw=config.dirman_password,
```

Comment:
This should be:

```
if promote:
    conn.connect(ccache)
else:
   connect as directory manager
```

variable `promote` should be already defined, and promote means domain level 1+ for now, so there is no need for gettting domain level one more time